Modules/Loadable/Markups/MRMLDM/vtkMRMLMarkupsDisplayableManager.h

```diff
@@ -138,8 +145,12 @@ class  VTK_SLICER_MARKUPS_MODULE_MRMLDISPLAYABLEMANAGER_EXPORT vtkMRMLMarkupsDis
   /// \sa IsManageable(vtkMRMLNode*), IsCorrectDisplayableManager()
   virtual bool IsManageable(const char* nodeClassName);
 
-  /// Contains class names of markups nodes that this displayable manager can handle
-  std::set<std::string> Focus;
+  /// Contains types of markups nodes that this displayable manager can
+  /// handle as well as the associated widgets.
+  std::map<std::string, vtkSlicerMarkupsWidget*> MarkupsNodesWidgets;
```

Comment:
Include in the variable name what is mapped to what. For example MarkupsTypeToWidget